orc8r/cloud/go/services/tenants/servicers/storage/storage_test.go

```diff
@@ -28,6 +28,10 @@ var (
 		Key:   "word",
 		Value: marshaledTenant0,
 	}
+
+	networkWildCardStr   = "*"
+	completeSearchResult = map[string]blobstore.Blobs{networkWildCardStr: {sampleTenant0Blob, sampleTenant1Blob}}
+	partialSearchResult  = map[string]blobstore.Blobs{networkWildCardStr: {sampleTenant0Blob}}
```

Comment:
Same as other comment -- scope these vars per-function, rather than in pkg-global state

I know this file already breaks that pattern, but let's not make it worse